oauth2client/service_account.py

```diff
@@ -108,7 +126,7 @@ def serialization_data(self):
     }
 
   def create_scoped_required(self):
-    return not self._scopes
+    return False
```

Comment:
See @anthmgoogle's response in an old commit. I believe I'll have to avoid commit-overwriting for the duration of this process...